modules/ROOT/pages/tools/neo4j-admin/neo4j-admin-import.adoc

```diff
@@ -619,6 +619,17 @@ Incremental import into an existing database.
 
 === Usage and limitations
 
+[WARNING]
+====
+The importer works well on standalone servers.
+
+In clustering environments with multiple copies of the database, the updated database *must be reseeded*.
```

Comment:
This is my wild guess, but I think we have to clarify that the database must be reseeded from a specific backup.

Reply:
I think you reseed the other databases from the one you have updated.

Reply:
So, the updated database must be used as a source for reseed?

Reply:
Yes, that's how I understand it. But @tinwelint can chime in here.

Reply:
Yes, the database you imported new data into is the only database that have the new data, right? So that's the one that has to be the seed source.